pyproject.toml

```diff
@@ -31,13 +30,15 @@ test = [
     "einops>=0.7.0",
     "protobuf>=4.23.4",
     "lightning-thunder==0.2.0.dev20240404; python_version >= '3.10'",
+    "litserve>=0.1.0" # litgpt.deploy
```

Comment:
You can assume that the test environment will install the "all" dependencies too

```suggestion
```